homeassistant/components/ollama/__init__.py

```diff
@@ -154,9 +159,11 @@ async def async_migrate_integration(hass: HomeAssistant) -> None:
             hass.config_entries.async_update_entry(
                 entry,
                 title=DEFAULT_NAME,
+                # Update parent entry to only keep URL, remove model
+                data={CONF_URL: entry.data[CONF_URL]},
```

Comment:
This means we need to bump to a new major version 3.1; attempting to downgrade from 2.3 to 2.2 will otherwise cause all sort of horrors caused by throwing KeyError when attempting to access `entry.data[CONF_MODEL]`